cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/aws/wizard-basic/management-credential-step/ManagementCredentialOneTime.test.tsx

```diff
@@ -0,0 +1,158 @@
+// React imports
+import React from 'react';
+
+// Library imports
+import { fireEvent, render, screen } from '@testing-library/react';
+import ManagementCredentialOneTime from './ManagementCredentialOneTime';
+
+// App imports
+
+const regionsMock = [
+    'us-east-1',
+    'us-east-2',
+    'us-west-1',
+    'us-west-2',
+    'eu-central-1',
+    'eu-east-1',
+    'eu-east-2',
+    'ap-east-1',
+    'ap-south-1',
+    'ca-central-1',
+];
+describe('ManagementCredentialOneTime', () => {
+    const useFormMock = {
+        ...jest.requireActual('react-hook-form'),
+        useForm: () => ({
+            formState: {
+                errors: {},
+            },
+            setValue: jest.fn(),
+            register: jest.fn(),
+        }),
+    };
+    it('should render', async () => {
+        const view = render(
+            <ManagementCredentialOneTime
+                initialRegion={'us-east-1'}
+                handleInputChange={jest.fn()}
+                handleSelectRegion={jest.fn}
+                regions={regionsMock}
+                methods={useFormMock.useForm()}
+            />
+        );
+        expect(view).toBeDefined();
+    });
+    it('initial values', () => {
```

Comment:
nit: I like the names to indicate the expected behavior, so I would name this 'should render initial values' (or something like that)